### PR TITLE
feat(specs): add new BigCommerce source fields

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-1029

### Changes included:

This PR adds the new fields to the source spec for a BigCommerce integration

### CODE REVIEW

File: specs/ingestion/common/schemas/source.yml [hunk 1]
```diff
@@ -126,9 +126,27 @@ SourceBigCommerce:
...
+      items:
+        $ref: "#/BigCommerceChannel"
   required:
     - store_hash
 
+BigCommerceChannel:
+  type: object
+  additionalProperties: false
+  properties:
+    ID:
```
Comment (millotp) [inline specs/ingestion/common/schemas/source.yml:140]:
```suggestion
    id:
```

File: specs/ingestion/common/schemas/source.yml [hunk 2]
```diff
@@ -126,9 +126,27 @@ SourceBigCommerce:
...
+      type: integer
+      description: The ID of the bigcommerce channel.
+    currencies:
+      type: array
+      items:
+        type: string
+        description: The currency of the channel, in the code form (trigram string)
+        example: "usd"
```
Comment (millotp) [inline specs/ingestion/common/schemas/source.yml:148]: I think `id` is required

Comment (shortcuts) [reply specs/ingestion/common/schemas/source.yml:148]: great catch ty!

File: specs/ingestion/common/schemas/source.yml [hunk 3]
```diff
@@ -126,9 +126,27 @@ SourceBigCommerce:
     storeHash:
       type: string
       description: The store hash identifying the store the shopper is signing in to.
+    channels:
+      type: array
```
[the remark this answers is not in the document]
Comment (shortcuts) [reply specs/ingestion/common/schemas/source.yml:130]: oki it's on the array

File: specs/ingestion/common/schemas/source.yml [hunk 4]
```diff
@@ -126,9 +126,30 @@ SourceBigCommerce:
...
+  type: object
+  additionalProperties: false
+  properties:
+    id:
+      type: integer
+      description: The ID of the bigcommerce channel.
+    currencies:
+      type: array
+      description: An array of currencies for the given channel `ID`, a currency is a trigram string that represents the currency code.
+      example: ["usd","eur"]
```
Comment (millotp) [inline specs/ingestion/common/schemas/source.yml:146]: weird that the linter didn't catch that
```suggestion
      example: ['usd','eur']
```

Comment (shortcuts) [reply specs/ingestion/common/schemas/source.yml:146]: hmmmmmmmmmmmm true

**Review by millotp — APPROVED** [review]

**Pull request merged**